joomla/joomla-cms
# [4.0] Fix wrong info for search in fields

Pull Request for Issue # .

### Summary of Changes
In fields, there is no fieldname and no author.


### Testing Instructions
See the view com_fields 
![grafik](https://user-images.githubusercontent.com/1035262/111847114-8049dc80-8908-11eb-9e53-02c279804d9a.png)

Compare before and after patch.
